src/core/abstract_simulation_store.jl

```diff
@@ -16,4 +16,9 @@ abstract type SimulationStore end
 # - write_result!
 # - read_result!
 # - write_optimizer_stats!
-# - read_problem_optimizer_stats
+# - read_optimizer_stats
+# - get_dm_data
+# - get_em_data
```

Comment:
I didn't notice these called outside of the store files. As long as that is correct, we don't need to make them required interface methods.

Reply:
These are used in the simulation state and simulation update code.